index.js

```diff
@@ -49,11 +52,12 @@ module.exports = function(options) {
       'install-sw.js',
       'sw.js',
       'sww.js',
-      hookSW,
       'cache.html',
       'cache.js',
       'manifest.appcache'
     ];
+    FILES_TO_REMOVE.push.apply(FILES_TO_REMOVE, hookSW);
```

Comment:
I used "apply" instead of the spread operator to support older versions of node (the spread operator is supported starting node v5 according to the es6 table). Tell me if this is fine for you.